sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2IndexesResourceImpl.java

```diff
@@ -54,7 +52,8 @@ public Response getAllIndexesForTable(
       throw new WebApplicationException("tableName must be provided", Status.BAD_REQUEST);
     }
 
-    BridgeSchemaClient.create(blockingStub).findTable(keyspaceName, tableName);
+    // check that we're authorized for the table
+    bridge.getTable(keyspaceName, tableName);
```

Comment:
Should the `ResourceBase` class provide a `callWithTable` which actually accepts the Table object, so we don't call the `getTable` twice? Maybe I am reading wrong and this is not possible :relaxed: